tests/test_utils_gz.py

```diff
@@ -3,6 +3,9 @@
 
 from scrapy.utils.gz import gunzip, is_gzipped
 from scrapy.http import Response, Headers
+
+from w3lib.encoding import html_to_unicode
```

Comment:
Could you please move this import above scrapy imports? This is very minor, but scrapy tries to group imports into 3 groups: stdlib imports, blank line, external libraries, blank line, scrapy imports.